src/App.jsx

```diff
@@ -10,25 +10,34 @@ function App() {
     // 👇️ Get the number between min (inclusive) and max (inclusive)
     return Math.floor(Math.random() * (max - min + 1)) + min;
   }
-
+  const reset = () => {
+    setNum(0);
+    setNum2(0);
+    setScore(0);
+    y.style.display="none"
+    z.style.display="none"
+  };
   
   const handleClick = () => {
     setNum(randomNumberInRange(1, 6));
     setNum2(randomNumberInRange(1, 6));
     setScore(score+num+num2);
-    if(score==100){
-      alert("Congratulations, you won!");
-    };
-    if(score>100){
-      alert("You lose!");
-    };
+  };
+  if(score==100){
+    y.style.display="block"
+  };
+  if(score>100){
+    z.style.display="block"
   };
   return (
     <div>
       <div>dice 1: {num}</div>
       <div>dice 2: {num2}</div>
       <div>score= {score}</div>
+      <div id='y'>Congratulations! You won!</div>
+      <div id='z'>You lost. Better luck next time.</div>
```

Comment:
Wenn man mit react arbeitet, ist es besser nicht direkt die html Elemente zu manipulieren (also zum Beispiel den style zu ändern), sondern conditional rendering zu benutzen, in diesem Fall zum Beispiel so:

```suggestion
      {score === 100 && <div>Congratulations! You won!</div>}
      {score > 100 && <div>You lost. Better luck next time.</div>}
```